tests/protos/kitchen-sink.proto

```diff
@@ -7,6 +7,7 @@
 package some_package;
 
 import "tests/protos/options.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
do you need to import this for the test? Also it's currently missing in the repo.

You could just extend another message in the same file for testing purposes